track_inventory/inventory_tracking_management.py

```diff
@@ -0,0 +1,38 @@
+"""Write report inventory to a json or csv file"""
```

Comment:
Excellent; nice functions here and this module only contains items related to outputting to a file.